### DISCUSSION TITLE
Github Pages isn't hosting my organization's page

### DISCUSSION THREAD

**Discussion by amrsekilly** [community/community#22373] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I have uploaded the build of a Gatsby.js website to Github Pages that you can see in this repo <a href="https://github.com/Exsite-Studio/Exsite-Studio" rel="noopener nofollow ugc">https://github.com/Exsite-Studio/Exsite-Studio</a></p>
<p>And configured the pages settings in the repo as you can see in the screenshot attached<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/a/ba0a6a347f468e2d5be8cbd6a4bfaa4642cb30d5.png" data-download-href="/uploads/short-url/qxN7DKUGTAFooD40um00Kzdmdi5.png?dl=1" title="Screen Shot 2021-12-10 at 3.11.21 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4142276/181070612-5f09e828-4e76-4194-9a72-918804c2bea8.png" alt="Screen Shot 2021-12-10 at 3.11.21 PM" data-base62-sha1="qxN7DKUGTAFooD40um00Kzdmdi5" width="690" height="447" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/a/ba0a6a347f468e2d5be8cbd6a4bfaa4642cb30d5_2_690x447.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/a/ba0a6a347f468e2d5be8cbd6a4bfaa4642cb30d5_2_1035x670.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/a/ba0a6a347f468e2d5be8cbd6a4bfaa4642cb30d5_2_1380x894.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/a/ba0a6a347f468e2d5be8cbd6a4bfaa4642cb30d5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2021-12-10 at 3.11.21 PM</span><span class="informations">2382×1546 393 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>In the past, the website was working properly. But, right now it’s not even accessible at <a href="https://exsite-studio.github.io" rel="noopener nofollow ugc">exsite-studio.github.io</a></p>
<p>Any help would be appreciated!</p>
<p>Thanks!</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/e/6ef0d30dfab23019c776c661fe07a11d4fcd19c8.png" data-download-href="/uploads/short-url/fPqu75B5UGdVucAEKkF1DzIcKs8.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181070613-789af854-37bf-47c0-bd50-bc9ff79d1a42.png" alt="image" data-base62-sha1="fPqu75B5UGdVucAEKkF1DzIcKs8" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/e/6ef0d30dfab23019c776c661fe07a11d4fcd19c8_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/e/6ef0d30dfab23019c776c661fe07a11d4fcd19c8_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/e/6ef0d30dfab23019c776c661fe07a11d4fcd19c8_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/e/6ef0d30dfab23019c776c661fe07a11d4fcd19c8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 194 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I can access it, it will just take time when you update it,</p>

**Comment by jfullstackdev** [comment]
<p>take note also that at this time, it’s <code>http</code> not <code>https</code>,</p>

**Comment by amrsekilly** [comment]
<p>Thanks, <a class="mention" href="/jdevstatic">@jdevstatic</a>!</p>
<p>Also, the SSL certificate provisioning is taking too long, it has been in this state for the past week. Any idea how I can fix this issue?</p>
<p>Thanks!</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/d/bd99ed38a814820cae7f9d2f6571378fbebe6a09.png" data-download-href="/uploads/short-url/r3i1KwAeKzy4Wh7AsyUq7LGKSlb.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181070617-78065bf7-587d-43bc-89cb-9cfdacec0ac0.png" alt="image" data-base62-sha1="r3i1KwAeKzy4Wh7AsyUq7LGKSlb" width="690" height="212" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd99ed38a814820cae7f9d2f6571378fbebe6a09_2_690x212.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd99ed38a814820cae7f9d2f6571378fbebe6a09_2_1035x318.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/b/d/bd99ed38a814820cae7f9d2f6571378fbebe6a09.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/d/bd99ed38a814820cae7f9d2f6571378fbebe6a09_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1180×364 26 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>is this correct information? <code>namecheap.com</code> also issues the certificate, so I think, they are the one responsible for that.</p>

**Comment by jfullstackdev** [comment]
<p>the other factor is where you are hosting your domain, like, is it AWS? or any other hosting platform, because from WHOIS, the domain is already registered</p>

**Comment by amrsekilly** [comment]
<p>I bought the domain from NameCheap and configured the DNS records to point to Github pages, in the past Github Pages were providing the SSL certificate. But right now, it’s showing this pending state for the SSL certificate. <a class="mention" href="/jdevstatic">@jdevstatic</a></p>

**Comment by amrsekilly** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/0/3091b80efd6985c2a522c59cd79640bbea51e0c7.png" data-download-href="/uploads/short-url/6VF7ru8DupLFzW05SI21wPBKG0v.png?dl=1" title="Screen Shot 2021-12-10 at 3.36.22 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4142276/181070618-1093b120-00f5-4468-bbc8-6b61953dd821.png" alt="Screen Shot 2021-12-10 at 3.36.22 PM" data-base62-sha1="6VF7ru8DupLFzW05SI21wPBKG0v" width="690" height="313" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/3091b80efd6985c2a522c59cd79640bbea51e0c7_2_690x313.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/0/3091b80efd6985c2a522c59cd79640bbea51e0c7_2_1035x469.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/0/3091b80efd6985c2a522c59cd79640bbea51e0c7_2_1380x626.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/3091b80efd6985c2a522c59cd79640bbea51e0c7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2021-12-10 at 3.36.22 PM</span><span class="informations">2358×1072 146 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><p></p>
<p>These are my DNS records on NameCheap <a class="mention" href="/jdevstatic">@jdevstatic</a></p>

**Comment by jfullstackdev** [comment]
<p>take a look at this :</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/a/ea3bce2f86bd0af31ab19bdb76e2c13109262230.png" data-download-href="/uploads/short-url/xq7Rm6TkDrlDb8QVNnxIpBdMfJu.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181070620-5127c832-60d6-4173-b34c-6a8f32243323.png" alt="image" data-base62-sha1="xq7Rm6TkDrlDb8QVNnxIpBdMfJu" width="689" height="457" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/a/ea3bce2f86bd0af31ab19bdb76e2c13109262230_2_689x457.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/a/ea3bce2f86bd0af31ab19bdb76e2c13109262230_2_1033x685.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/a/ea3bce2f86bd0af31ab19bdb76e2c13109262230.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/a/ea3bce2f86bd0af31ab19bdb76e2c13109262230_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1292×856 145 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>the complete issue is here</p>
<aside class="onebox githubissue" data-onebox-src="https://github.com/isaacs/github/issues/1675">
  <header class="source">

      <a href="https://github.com/isaacs/github/issues/1675" target="_blank" rel="noopener nofollow ugc">github.com/isaacs/github</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/isaacs/github/issues/1675" target="_blank" rel="noopener nofollow ugc">GitHub Pages: Generate SSL certificate for www subdomain when a custom domain is set to an apex (and vice versa)</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2019-10-11" data-time="15:28:00" data-timezone="UTC">03:28PM - 11 Oct 19 UTC</span>
      </div>

        <div class="date">
          closed <span class="discourse-local-date" data-format="ll" data-date="2021-07-20" data-time="22:11:19" data-timezone="UTC">10:11PM - 20 Jul 21 UTC</span>
        </div>

      <div class="user">
        <a href="https://github.com/jakejarvis" target="_blank" rel="noopener nofollow ugc">
          <img alt="jakejarvis" src="https://user-images.githubusercontent.com/47092464/181070621-dfc3f3d8-ac30-4802-9e36-bf3168d44bed.jpeg" class="onebox-avatar-inline" width="20" height="20">
          jakejarvis
        </a>
      </div>
    </div>

    <div class="labels">
        <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">
          pages
        </span>
    </div>
  </div>
</div>

  <div class="github-row">
    <p class="github-body-container">There's a _loooooong_ discussion over at https://github.community/t5/GitHub-Page<span class="show-more-container"><a href="" rel="noopener" class="show-more">…</a></span><span class="excerpt hidden">s/Does-GitHub-Pages-Support-HTTPS-for-www-and-subdomains/td-p/7116 about this — incorrectly marked as "solved," of course.

When setting a custom domain on a GitHub Pages repo to a root domain, like `example.com`, a Let's Encrypt certificate for it is generated when Enforce HTTPS is enabled (obviously). But there is absolutely no way to have `www.example.com` redirect to the root/apex domain `example.com` with a valid SSL certificate as well.

It absolutely sucks having to add yet _another_ service into the mix just to get https://www.example.com to redirect to https://example.com without showing a certificate warning. It seems like a pretty safe assumption that if someone enters an apex domain then they'll want www to mirror it/redirect to it — wanting different content at @ and www is a _super_ rare edge case (and probably accidental if it does occur, to be honest). 

Certain domain registrars and/or DNS providers can do this for free (Google Domains, Cloudflare, etc.) but I'd love to keep things consistent re: hosting...and, of course, my personal DNS provider and many others charge extra for this. 

Please add `www` as an alt name on the LE certificate by default if the user inputs an apex domain as the custom domain!</span></p>
  </div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/c/ec48b0105d2296d451e9dff73d2d1a163a2a760b.png" data-download-href="/uploads/short-url/xIgpMnvPXNy58QH1G7fdeJN2Zjt.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181070622-c0c6c1fb-92f9-4b4e-bfaf-f3d109aca6c2.png" alt="image" data-base62-sha1="xIgpMnvPXNy58QH1G7fdeJN2Zjt" width="690" height="216" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/c/ec48b0105d2296d451e9dff73d2d1a163a2a760b_2_690x216.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/c/ec48b0105d2296d451e9dff73d2d1a163a2a760b_2_1035x324.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/c/ec48b0105d2296d451e9dff73d2d1a163a2a760b_2_1380x432.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/c/ec48b0105d2296d451e9dff73d2d1a163a2a760b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1406×442 51.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>and I think, if ever it’s still not working, you can ping him, he is a moderator here in GitHub Community who has the direct access to the backend,</p>

**Comment by amrsekilly** [comment]
<p>Thanks <a class="mention" href="/jdevstatic">@jdevstatic</a> I already configured the www CNAME, but I’m still waiting for the TLS Certificate provisioning</p>

**Comment by yoannchaudet** [comment]
<p>Hello <a class="mention" href="/amrsekilly">@amrsekilly</a>!</p>
<p>Your certificate has not been provisioned yet because your DNS is improperly configured.</p>
<p>You will need to remove the record for <code>162.255.119.208</code>. This is not a Pages IP.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/b/7b3ddf66c24d646a7195f0f88d74de3255bb874e.png" data-download-href="/uploads/short-url/hAfi0pZaE3ndQiBgRUSeRWOcz8i.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/14911070/181070623-4c1ad518-68f1-47de-8f50-c169a188c49c.png" alt="image" data-base62-sha1="hAfi0pZaE3ndQiBgRUSeRWOcz8i" width="690" height="402" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7b3ddf66c24d646a7195f0f88d74de3255bb874e_2_690x402.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7b3ddf66c24d646a7195f0f88d74de3255bb874e_2_1035x603.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7b3ddf66c24d646a7195f0f88d74de3255bb874e_2_1380x804.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7b3ddf66c24d646a7195f0f88d74de3255bb874e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1424×830 102 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Once fixed, your TLS certificate should provision just fine.</p>

**Comment by amrsekilly** [comment]
<p>Thank you so much, <span class="mention">@yoannchaude</span>!! That fixed the issue!</p>
